compatibility/krastorio2/data-updates.lua

```diff
@@ -31,78 +28,114 @@ util.add_research_ingredient("fluid-handling", "logistic-science-pack")
+data.raw.item["imersium-plate"].stack_size = 100
+data.raw.item["enriched-iron"].stack_size = 100
+data.raw.item["enriched-copper"].stack_size = 100
+data.raw.item["enriched-rare-metals"].stack_size = 100
+data.raw.item["lithium-chloride"].stack_size = 100
+data.raw.item["lithium"].stack_size = 100
+data.raw.item["sulfur"].stack_size = 100
+data.raw.item["lithium-sulfur-battery"].stack_size = 100
+data.raw.item["explosives"].stack_size = 100
+data.raw.item["imersite-powder"].stack_size = 100
+data.raw.item["tritium"].stack_size = 100
+data.raw.item["fuel"].stack_size = 50
+data.raw.item["bio-fuel"].stack_size = 50
+data.raw.item["advanced-fuel"].stack_size = 50
+data.raw.item["iron-stick"].stack_size = 200
+data.raw.item["iron-gear-wheel"].stack_size = 200
+data.raw.item["rocket-control-unit"].stack_size = 50
+data.raw.item["low-density-structure"].stack_size = 50
+data.raw.item["rocket-fuel"].stack_size = 50
+data.raw.item["matter-stabilizer"].stack_size = 50
+data.raw.item["charged-matter-stabilizer"].stack_size = 50
+data.raw.item["processing-unit"].stack_size = 200
+data.raw.item["engine-unit"].stack_size = 200
+data.raw.item["electric-engine-unit"].stack_size = 200
+data.raw.item["flying-robot-frame"].stack_size = 200
+data.raw.item["imersite-crystal"].stack_size = 200
+data.raw.item["uranium-235"].stack_size = 200
+data.raw.item["uranium-238"].stack_size = 200
+data.raw.item["energy-control-unit"].stack_size = 200
+data.raw.item["space-research-data"].stack_size = 200
+data.raw.item["kr-black-reinforced-plate"].stack_size = 100
+data.raw.item["kr-white-reinforced-plate"].stack_size = 100
+data.raw.item["kr-steel-pipe"].stack_size = 100
+data.raw.ammo["explosive-turret-rocket"].stack_size = 50
+data.raw.ammo["nuclear-turret-rocket"].stack_size = 50
+data.raw.ammo["antimatter-turret-rocket"].stack_size = 50
+data.raw.ammo["artillery-shell"].stack_size = 50
+data.raw.ammo["nuclear-artillery-shell"].stack_size = 50
+data.raw.ammo["antimatter-artillery-shell"].stack_size = 50
+data.raw.ammo["atomic-bomb"].stack_size = 50
+data.raw.ammo["antimatter-rocket"].stack_size = 50
 
--- change resources autoplace
+if mods["PlutoniumEnergy"] then
+  data.raw.item["plutonium-238"].stack_size = 200
+  data.raw.item["plutonium-239"].stack_size = 200
+  data.raw.ammo["plutonium-atomic-bomb"].stack_size = 50
+end
+
+-- Add containers
+data.raw.ammo["artillery-shell"].ic_create_container = true
+data.raw.ammo["atomic-bomb"].ic_create_container = true
+data.raw.ammo["antimatter-rocket"].ic_create_container = true
+
+if mods["PlutoniumEnergy"] then 
+  data.raw.ammo["plutonium-atomic-bomb"].ic_create_container = true
+end
+
+-- Revert fuels' category to adv.chemical
+data.raw.item["rocket-fuel"].fuel_category = "advanced-chemical"
+
+-- Add chemical fuel back to boat (but NOT adv. chamical)
+data.raw.car["indep-boat"].burner.fuel_category = nil
+data.raw.car["indep-boat"].burner.fuel_categories = { "chemical", "vehicle-fuel" }
```

Comment:
Ideally these would use table.insert rather than overwriting them (as in fuel-categories.lua) but not a big concern, and it is a pain to do with the `fuel_category/fuel_categories` split.